app/models/spree_avatax/return_invoice.rb

```diff
@@ -31,7 +31,7 @@ class << self
     # After the reimbursement completes the ".finalize" method will get called and we'll commit the
     #   return invoice.
     def generate(reimbursement)
-      if !SpreeAvatax::Config.enabled
+      if !SpreeAvatax::Config.enabled || !reimbursement.order.store.country.iso =~ /(US|CA)/
```

Comment:
Maybe make this a method on the order? `north_american_order?`. Can we rely on the `store_id` on the order as an alternative?

Reply:
We can't rely on the store being assigned to a specific country since country is a validation on the store, however the store is a validation on an order. Do we want to define that order instance method here or in the host app? We could also use the store_id's. Those are highly unlikely to change.

Reply:
Actually... store_id's might be fine.

Reply:
Magic numbers are always going to be a smell due to brittleness, even if they're unlikely to change in our specific instance.

I think I might have checked whether the tax-adjustments on the original order were Avatax-calculated:

```
reimbursement.order.all_adjustments.tax.all?(&:avatax?)
```